css/application-styles.css

```diff
@@ -0,0 +1,226 @@
+.main-nav {
+  position: static;
+}
+
+.paw-logo {
+  display: inline-block;
+}
+
+/* INPUTS AND LABELS STYLES START */
+input {
+  width:460px;
+  height: 64px;
+  border: 1px solid #D2D2D2;
+  border-radius: 3px;
+  padding: 0;
+  margin-bottom: 20px;
+}
```

Comment:
It is only my suggestion :) I've added validation and little bit of style on input and I guess since our forms are similar they can follow same styles. For example on focus I have: 

input:focus,
textarea:focus {
  background-color: #fff2f4;
}

And for validation:

input:invalid{
  border: 2px solid red;
}

As well the input text size is very small, in table booking form I've changed it to 32px ( added helper class f-s32in all inputs in html) Please advise if you agree on this size, if no, we can change to in both our forms to another one).

Reply:
Yes, Im going to do in mine also